modules/account-lib/src/coin/xrp/iface.ts

```diff
@@ -0,0 +1,33 @@
+export interface SignedTransaction {
+  signedTransaction: string;
+  id: string;
+}
+
+export interface ApiMemo {
+  MemoData?: string;
+  MemoType?: string;
+  MemoFormat?: string;
+}
+
+export interface TxJSON {
+  account: string;
+  type: string;
+  memos?: { Memo: ApiMemo }[];
+  flags?: number;
+  fulfillment?: string;
+  amount?: string;
+  destination?: string;
+  fee?: string;
+  sequence?: number;
```

Comment:
Please create a common interface for `_sequence` under [BaseCoin](https://github.com/BitGo/BitGoJS/blob/master/modules/account-lib/src/coin/baseCoin/iface.ts)
This field is common across other coins, although its name may be different (e.g. `counter`, `nonce`, `sequenceId`)
The interface could be something like:
```
interface sequenceId {
	name: string;    // “Nonce”, “Sequence Id”, “Counter”
	keyword: string; // “nonce”, “sequenceId”, “counter”
	value: string;
}
```